src/rsht.c

```diff
@@ -32,7 +35,7 @@ static unsigned long djb2(const unsigned char *str) {
 
 // bool: did it work?
 static bool setcapacity(rsht_ht *ht, const size_t capacity) {
-  rsht_entry *items = realloc(ht->items, capacity * sizeof(rsht_entry));
+  rsht_entry *items = (rsht_entry*)realloc(ht->items, capacity * sizeof(rsht_entry));
```

Comment:
I would prefer a space between types and the `*`, so that it looks similar to the declaration. It looks weird to me to cast something to `rsht_entry*` but store it in a `rsht_entry *`.

Additionally, could you put a space between casts and what they are casting?